core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_autocomplete.scss

```diff
@@ -13,7 +13,13 @@
 
     .p-autocomplete-input {
         all: unset;
+        border: none;
+        font-family: $font-default;
+        font-size: $font-size-md;
```

Comment:
isn't this being inherited from the input text field?

Reply:
yes but I have to do and `all: unset` to reset the states and almost everything from the input, because the input in this case is actually hidden, so It does not need any styles besides fonts and height